IBM/carbon-addons-iot-react
# fix(rowactioncell): incorrect colors

In reference to #1346 
This slightly changes the fix to #1314 

**Summary**

RowActionCell will continue to use `hasIconOnly` with the tooltip being positioned to the left
OverflowMenu will stay gray20 on hover instead of blue


**Change List (commits, features, bugs, etc)**

- Fixed invalid `null` props in ListItem stories
- Removed custom styling that was turning icons blue in row actions as it doesn't adhere to the design guidelines
- Positioned the rowActionCell tooltip to the left
- Made the RowActionCell button small to match the size of the OverflowMenu


**Acceptance Test (how to verify the PR)**

- Go to Table row actions story and see that the icon is still the correct color with the tooltip to the left
- Go to TableBodyRow actions stories and see that the icon is still the correct color with the tooltip to the left

![Screen Shot 2020-06-29 at 12 18 40 PM](https://user-images.githubusercontent.com/25177649/86037603-3b41eb00-ba05-11ea-969c-9c913d22bd37.png)
![Screen Shot 2020-06-29 at 12 16 29 PM](https://user-images.githubusercontent.com/25177649/86037605-3bda8180-ba05-11ea-935b-d614e6b79866.png)
![Screen Shot 2020-06-29 at 12 16 25 PM](https://user-images.githubusercontent.com/25177649/86037607-3bda8180-ba05-11ea-8be3-489c06880733.png)
